Update customize-your-organization-theme.md (#490)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/setup/customize-your-organization-theme.md

    
    ![Tap or click Edit to change your theme](../media/f777ecde-b165-408b-b9de-d8bc8fb82c89.png)
  

> 5. Change the theme elements you want for your organization, your changes will be previewed on this page:

Reviewer: ```suggestion
5. Change the theme elements you want for your organization. Your changes will be previewed on this page:
```

Reviewer 2: what page?

>   - **Logo image**: Choose whether to use an image from a URL or to upload an image. If using a URL, please ensure that the URL uses HTTPS and is 200 x 30 pixels of any format of any size. You may upload a logo that is 200 x 30 pixels in JPG, PNG, GIF, or SVG format under 10kb.

Reviewer: ```suggestion
  - **Logo image**: Choose whether to use an image from a URL or to upload an image. If you are using a URL, please ensure that the URL uses HTTPS and is 200 x 30 pixels of any format of any size. You may upload a logo under 10 KB that is 200 x 30 pixels in JPG, PNG, GIF, or SVG format .
```

>   - **URL for a clickable logo**: You may use your logo in the navigation bar as a link to any company resource. Here you may enter the URL for the logo, starting with http:// or https://. This is optional.

Reviewer: ```suggestion
  - **URL for a clickable logo**: You may use your logo in the navigation bar as a link to any company resource. You may enter the URL for the logo here, starting with http:// or https://. This is optional.
```

>   - **URL for a clickable logo**: You may use your logo in the navigation bar as a link to any company resource. Here you may enter the URL for the logo, starting with http:// or https://. This is optional.

Reviewer 2: ```suggestion
  - **URL for a clickable logo**: You can use your logo in the navigation bar as a link to any company resource. Enter the URL for the logo, starting with http:// or https://. This is optional.
```

  
  - **Prevent users from overriding theme**: Flip this toggle to prevent users from choosing their own theme from our theme selection. This does not prevent users from being able to set a high contrast theme.
    
  - **Nav bar background color**: Select a color to use for the background of the navigation bar. The appears at the top on every page.
    
  - **Text and icons**: Select a color to use for the text and icons on the top navigation bar.
    

>   - **Accent color**: Select a color to use for the, nav bar button hover color, and page accents like buttons and text on certain applications.

Reviewer: ```suggestion
  - **Accent color**: Select a color to use for the navigation bar button hover color and page accents like buttons and text on certain applications.
```

>   - **User name**: Choose whether or not to show a user's full name at the entrypoint to the account manager in the top right of the page when they are signed in. By default, users will see their photo or their initials if a photo hasn't been uploaded.

Reviewer: ```suggestion
  - **User name**: Choose whether to show a user's full name at the entry point to the account manager in the top right of the page when the user is signed in. By default, users will see their photo or their initials if a photo hasn't been uploaded.
```

    
6. Click or tap **Save**.
    
You'll see your new theme on the Microsoft 365 admin center right away and after a short delay, you'll see it throughout Office 365, including Outlook and SharePoint pages. You can remove your custom icon or custom colors at any time. Just return to the theme page and choose **Remove custom theming**
  
## Best Practices

> When choosing a **Logo image**. We recommend using the SVG file type, if you can, so that your logo will look high resolution on all screens and all zoom levels.

Reviewer: ```suggestion
When choosing a **Logo image**, we recommend using an SVG file type, wherever possible, so that your logo will have a high resolution appearance on all screens and at all zoom levels.
```

> When choosing custom colors, pick a **Nav bar background color** that is a high contrast ratio with the **Logo image** that you picked. Also, pick a **Text and icons** color with a high contrast ratio to the **Nav bar background color** to ensure that all text and icons are easily visible.

Reviewer: ```suggestion
When choosing custom colors, choose a **Nav bar background color** that has a high contrast ratio with the **Logo image** that you picked. Also choose a **Text and icons** color with a high contrast ratio to the **Nav bar background color** to ensure that all text and icons are easily visible.
```

When choosing custom colors, pick an **Accent color** that shows up well on a white or light background. The **Accent color** is used to color some links and buttons that show up on a white or light background. For example, the **Accent color** is used to color elements in a user's inbox and on their Office.com portal page. 
  
The recommended contrast ratio between text, icon, or button color and background color is 4.5:1.

> Here is a simple flow chart to help you quickly get set up with a visually appealing theme for your custom Office 365 theme for your organization:

Reviewer: ```suggestion
Here is a simple flow chart to help you quickly get set up with a visually appealing custom Office 365 theme for your organization:
```

  - I would like to use a colorful version of our logo.
    - We recommend the following settings:
      - **Logo image**: Your organization's colorful logo.

>       - **Navigation bar color**: A neutral color. we recommend #FAF9F7 for a light color and #252423 for a dark color.

Reviewer: ```suggestion
      - **Navigation bar color**: A neutral color. We recommend #FAF9F7 for a light color and #252423 for a dark color.
```

      - **Text and icon color**: A color to contrast the **Navigation bar color**. We recommend #FAF9F7 for a light color and #252423 for a dark color.

>       - **Accent color**: A dark brand color. This color carries down to certain applications and must be visible on a light background.

Reviewer: ```suggestion
      - **Accent color**: A dark shaded color. With certain applications, this color must be visible on a light background.
```

  - I would like to use a neutral version of our logo and represent color in the navigation bar.
    - We recommend the following settings:
      - **Logo image**: Your organization's neutral logo.
      - **Navigation bar color**: A brand color that contrasts against your logo.

>       - **Text and icon color**: Choose a color that contrasts against the brand color you chose for the **Navigation bar color**. We recommend #252423 for a dark color and #FAF9F7 for a light color.

Reviewer: ```suggestion
      - **Text and icon color**: Choose a color that contrasts against the brand color you chose for the **Navigation bar color**. We recommend #252423 for a dark color and #FAF9F7 for a light color.
```

>       - **Accent color**: A dark brand color. This color carries down to certain applications and must be visible on a light background.

Reviewer: ```suggestion
      - **Accent color**: A dark brand color. With certain applications, this color must be visible on a light background.
```
